lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -145,7 +145,8 @@ def _get_context():
     context["csfb_mcc"] = _get_csfb_mcc()
     context["csfb_mnc"] = _get_csfb_mnc()
     context["lac"] = _get_lac()
-    context["use_stateless"] = get_service_config_value("mme", "use_stateless", "")
+    use_stateless = get_service_config_value("mme", "use_stateless", False)
+    context["use_stateless"] = "" if use_stateless == False else use_stateless    
```

Comment:
nit: can you do

```
context[...] = use_stateless or ''
```

Reply:
Thank you for reviewing and advising this PR.
I tried to change the code style based on your advice.

`context["use_stateless"] = get_service_config_value("mme", "use_stateless", "") or ""`